src/sentry/models/groupsubscription.py

```diff
@@ -6,18 +6,28 @@
 from django.utils import timezone
 
 from sentry.db.models import (
-    BoundedPositiveIntegerField, FlexibleForeignKey, Model, BaseManager,
+    BaseManager, BoundedPositiveIntegerField, FlexibleForeignKey, Model,
     sane_repr
 )
 
 
 class GroupSubscriptionReason(object):
+    implicit = -1  # not for use as a persisted field value
     unknown = 0
     comment = 1
     assigned = 2
     bookmark = 3
     status_change = 4
 
+    descriptions = {
+        implicit: u"have opted to receive updates for all issues within "
+                  "projects that you are a member of",
```

Comment:
I couldn't think of a good way to word this, this explains accurately but reads a little weird, suggestions accepted.